pkgs/top-level/python-packages.nix

```diff
@@ -653,6 +653,8 @@ in {
 
   databricks-cli = callPackage ../development/python-modules/databricks-cli { };
 
+  databricks-connect = callPackage ../development/python-modules/databricks-connect { };
```

Comment:
```suggestion
  databricks-connect = callPackage ../development/python-modules/databricks-connect {
    inherit (pkgs) jdk;
  };
```

Reply:
What is the reason for this change?

I have added your suggestion (see commit 44e9626), but had to edit databricks-connect/default.nix to avoid the following build error:
`error: anonymous function at /home/kristoffer/Desktop/tmp/nixpkgs/pkgs/development/python-modules/databricks-connect/default.nix:1:1 called with unexpected argument 'jdk', at /home/kristoffer/Desktop/tmp/nixpkgs/lib/customisation.nix:69:16`

Reply:
sorry, thought it was what you had before, but it was probably:
```
    jdk = pkgs.openjdk;
```

Reply:
and the reasoning behind it is that someone could add ` python3Packages.jdk`, and then `callPackage` will bring in that package, rather than the intended `pkgs.openjdk` or what was menat to be referenced.